examples/embedded/src/main/java/org/eclipse/jetty/embedded/OneServletContextJmxStats.java

```diff
@@ -45,7 +45,7 @@ public static Server createServer(int port)
         context.addServlet(DefaultServlet.class, "/");
 
         // Add Connector Statistics tracking to all connectors
-        ServerConnectionStatistics.addToAllConnectors(server);
+        server.addToAllConnectors(new ConnectionStatistics());
```

Comment:
maybe `addBeanToAllConnectors()`

Reply:
The method currently takes a `Connection.Listener` so what about `addListenerToAllConnectors()`? 
Or should I generalise it to `addBeanToAllConnectors(Object o)`.

Reply:
I think the impl just does an addBean, so `addBeanToAllConnectors(Object)` is best.   If I missed something and it is about `Listeners`, then `addListenerToAllConnectors(Listener)